mleap-spark-testkit/src/main/scala/org/apache/spark/ml/parity/SparkParityBase.scala

```diff
@@ -66,12 +75,7 @@ abstract class SparkParityBase extends FunSpec with BeforeAndAfterAll {
   val dataset: DataFrame
   val sparkTransformer: Transformer
 
-  lazy val spark = SparkSession.builder().
-    appName("Spark/MLeap Parity Tests").
-    master("local[2]").
-    getOrCreate()
-
-  override protected def afterAll(): Unit = spark.stop()
+  val spark = SparkEnv.spark
```

Comment:
Use a global spark context across all suites. Previous test shutdown spark context when every suite finished, but the spark context variable are referred in many places which cause error like: `SparkContext has been shutdown`
In mleap spark related tests, all tests using the same config spark context, so we can use a global one instead.

Reply:
++ for having a single spark session